domain/schema/model/sql/0017-machine-cloud-instance.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE machine_cloud_instance (
     machine_uuid TEXT NOT NULL PRIMARY KEY,
-    instance_id TEXT NOT NULL,
-    display_name TEXT NOT NULL,
```

Comment:
todo: Add a comment why `instance_id` can be NULL. Feels like a wart. `display_name` is definitely optional.

Reply:
I'll add a comment, but read the description 😉